build.sbt

```
@@ -1,25 +1,27 @@
 name := "sangria-play-json"
 organization := "org.sangria-graphql"
-version := "1.0.6-SNAPSHOT"
+mimaPreviousArtifacts := Set("org.sangria-graphql" %% "sangria-play-json" % "1.0.5")
 
 description := "Sangria play-json marshalling"
 homepage := Some(url("http://sangria-graphql.org"))
 licenses := Seq("Apache License, ASL Version 2.0" → url("http://www.apache.org/licenses/LICENSE-2.0"))
 
-scalaVersion := "2.12.7"
-crossScalaVersions := Seq("2.11.11", "2.12.7")
+scalaVersion := "2.13.0"
+crossScalaVersions := Seq("2.11.12", "2.12.10", scalaVersion.value)
 
 scalacOptions ++= Seq("-deprecation", "-feature")
 
 libraryDependencies ++= Seq(
-  "org.sangria-graphql" %% "sangria-marshalling-api" % "1.0.3",
-  "com.typesafe.play" %% "play-json" % "2.6.10",
+  "org.sangria-graphql" %% "sangria-marshalling-api" % "1.0.4",
+  "com.typesafe.play" %% "play-json" % "2.7.4",
 
-  "org.sangria-graphql" %% "sangria-marshalling-testkit" % "1.0.1" % "test",
-  "org.scalatest" %% "scalatest" % "3.0.5" % "test")
+  "org.sangria-graphql" %% "sangria-marshalling-testkit" % "1.0.2" % "test",
+  "org.scalatest" %% "scalatest" % "3.0.8" % "test")
```

Little possible improvement: using `Test` instead of `"test"`.
```suggestion
  "org.scalatest" %% "scalatest" % "3.0.8" % Test)
```